tests/kafkatest/services/connect.py

```diff
@@ -37,7 +53,7 @@ def __init__(self, context, num_nodes, kafka, files):
     def pids(self, node):
         """Return process ids for Kafka Connect processes."""
         try:
-            return [pid for pid in node.account.ssh_capture("cat /mnt/connect.pid", callback=int)]
+            return [pid for pid in node.account.ssh_capture("cat " + self.PID_FILE, callback=int)]
```

Comment:
pid files are good :) However `clean_node` probably should not rely on the pid file